src/device-registry/utils/get-measurements.js

```diff
@@ -19,6 +20,60 @@ const { generateDateFormat, generateDateFormatWithoutHrs } = require("./date");
 
 const getDetail = require("../utils/get-device-details");
 
+const isRecentTrue = (recent) => {
+  let isRecentEmpty = isEmpty(recent);
+  if (isRecentEmpty == true) {
+    return true;
+  }
+  if (recent.toLowerCase() == "yes" && isRecentEmpty == false) {
```

Comment:
Hi @ja-odur 
just reminding us that we shall have to make appropriate changes in the frontend after this gets merged into staging. Key thing to note is that recent query parameter now takes strings **yes** or **no**....something which is different from the previous expected values of either **true** or **false**.